addons/website_slides/static/src/js/slides_course_slides_list.js

```diff
@@ -1,251 +1,116 @@
-                }
-                target.removeClass('slide-hovered');
-            });
-            target.on('drop', function (ev){
-                if(ev.preventDefault){
-                    ev.preventDefault();
-                }
-                if(self.draggedElement.hasClass('section-draggable')  && target.hasClass('section-draggable')){
-                    target.removeClass('slide-hovered');
-                    target.removeClass('hold');
-                    self.dropTarget = $(ev.currentTarget);
-                    if(target !== self.draggedElement && $(ev.currentTarget).hasClass('section-draggable')){
-                        self.draggedElement[0].parentNode.removeChild(self.draggedElement[0]);
-                        var dropHTML = ev.originalEvent.dataTransfer.getData('text/html');
-                        target[0].insertAdjacentHTML('beforebegin',dropHTML);
-                        self.draggedElement = $(target[0].previousSibling);
-                        self._reorderCategories();
-                        self._reorderSlides();
-                        self._rebindUploadButton(self.draggedElement.attr('category_id'));
-                    }
-                    self._unbindAll();
-                    self._bindEvents();
-                }
-            });
-            target.on('dragend', function (ev){
-                if (ev.preventDefault){
-                    ev.preventDefault();
-                }
-                target.removeClass('slide-hovered');
-                target.removeClass('hold');
-            });
-        },
-        _reorderCategories: function (){
-            var self = this;
-            self._getCategories();
-            self._rpc({
-                route: '/web/dataset/resequence',
-                params: {
-                    model: "slide.category",
-                    ids: self.categories
-                }
-            }).then(function (){
-                self._resetCategoriesIndex();
-            });
-        },
-        _resetCategoriesIndex: function (){
-            var categoriesIndexes = $('.section-index')
-            for (var i = 0; i < categoriesIndexes.length; i++){
-                $(categoriesIndexes[i]).text(i+1)
+'use strict';
+
+var sAnimations = require('website.content.snippets.animation');
+
+sAnimations.registry.websiteSlidesCourseSlidesList = sAnimations.Class.extend({
+    selector: '.o_wslides_course_slides_list',
+    xmlDependencies: ['/website_slides/static/src/xml/website_slides_upload.xml'],
+
+    start: function () {
+        this._super.apply(this,arguments);
+
+        this._updateHref();
+        this._bindSortable();
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * Change links href to fullscreen mode for SEO
+     *
+     * @private
+     */
+    _updateHref: function () {
```

Comment:
Small comment is welcome :dancing_women: 

link-to-slide -> o_wslides_slides_list_href ?

Reply:
Done.

Reply:
(to "o_wslides_slides_list_slide_link")